model/operation.cc

```diff
@@ -921,37 +1002,49 @@ namespace minsky
 
   template <> void Operation<OperationType::le>::iconDraw(cairo_t* cairo) const
   {
+	float sf = scaleFactor(); 	     
```

Comment:
In these cases, declaring sf double would be better. cairo_scale takes double arguments.

Reply:
Done :)